extensions/amp-sidebar/1.0/sidebar.js

```diff
@@ -146,92 +159,118 @@ function SidebarWithRef(
   useLayoutEffect(() => {
     const sidebarElement = sidebarRef.current;
     const backdropElement = backdropRef.current;
-    if (!sidebarElement || !backdropElement) {
+    if (!sidebarElement || !backdropElement || !side) {
```

Comment:
This hook now is too complex and we should move it to a separate file, e.g. as `useSidebarAnimation(open, ...)`. This would include the refs above as well.

Reply:
Thanks, updated in the latest.